src/data/localization.rs

```diff
@@ -285,6 +285,41 @@ impl Localization {
         }
     }
 
+    #[cfg(feature = "ck3")]
+    pub fn verify_name_exists(&self, name: &Token, max_sev: Severity) {
+        if name.as_str().is_empty() {
+            return;
```

Comment:
Should empty name return an error?

Reply:
Yeah probably. I copied this from the more general verify check, where it makes more sense.